peg/elm_post.h

```diff
@@ -0,0 +1,155 @@
+/*
+ *   Copyright (c) 2022 Nik Silver
+ *   based on the Kotlin code by Jan Dolinár
+ *
+ *   This source code is released for free distribution under the terms of the
+ *   GNU General Public License version 2 or (at your option) any later version.
+ *
+ *   This module contains functions to generate tags for Elm.
+ */
```

Comment:
Could you put the URL for a reference of the target language as Jan did in kotlin.peg?